tools/style-check.sh

```diff
@@ -3,13 +3,9 @@
 # Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
 # This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2021-Present Datadog, Inc.
 
-### Set directory names
-CURRENTDIR=$PWD
-SCRIPTPATH=$(readlink -f "$0")
-SCRIPTDIR=$(dirname $SCRIPTPATH)
-cd $SCRIPTDIR/../
-TOP_LVL_DIR=$PWD
-cd $CURRENTDIR
+set -euo pipefail
+
+cd "$(git rev-parse --show-toplevel)"
```

Comment:
minor This does not work well in some cases (worktrees / copies of the directory)

Reply:
Good point, I will change that

Reply:
It should work with worktrees though

Reply:
Done